src/adapter.ts

```diff
@@ -393,69 +390,88 @@ export class SequelizeAdapter<
       return result;
     }
 
-    await this._get(id, {
+    const instance = await this._get(id, {
       ...params,
-      query: {
-        ...params?.query,
-        $select: [this.id]
-      }
-    });
+      sequelize: { ...params.sequelize, raw: false }
+    }) as Model;
 
-    try {
```

Comment:
I don't like `instance.update` much. It adds magic for `paranoid` and virtual fields. I think it only should be used when `$select: undefined`. As soon as `$select !== undefined` it's hard to predict what happens for complex code bases.